docs/advanced/storageclass.md

```diff
@@ -44,6 +44,12 @@ The number of replicas created for each volume in Longhorn. Defaults to `3`.
 
 ![](/img/v1.2/storageclass/create_storageclasses_replicas.png)
 
+:::info important
+
+When the value is `1`, the created volume from this StorageClass has only one replica, it may block the [Node Maintenance](../host/host.md#node-maintenance), check the section [Single-Replica Volumes](../troubleshooting/host.md#single-replica-volumes) and set a proper global option.
```

Comment:
**Number of Replicas** is a global setting
==
It only affects the volumes created from this `StorageClass`.

I guess your comment has a wrong assumption.

User can create many `StorageCalss`es, and when creating `Volume` he can/may select which `StorageClass` to based on. Only those `StorageCalss`es which have `replica count` 1 matter.

Reply:
The first part of your draft mentions a replica count value of 1. The last part tells the user to "set a proper global option". 

So the global option that you mentioned was the StorageClass itself and not the replica count? You want users to specify which StorageClass will be used to create volumes by default? If this is correct, include these details in the note.

Reply:
Seems some concepts are mixed.

(1) [StorageClass](https://docs.harvesterhci.io/v1.3/advanced/storageclass) is like a model, user can creates as many as he wants;
(2) When create [volume](https://docs.harvesterhci.io/v1.3/volume/index#create-an-empty-volume), user can refer any of his StorageClass; the parameters from SC are converted to paramters of volume

(3) When LH sees the volumes, it will do many tasks, including create the expected number of `replicas` for this volume.
(4) Replicas are scheduled to a target node
(5) When node `maintenance` happens, the `replica` on this node will affect if this node can be successfully drained.
(6) To give user more flexibility, LH has a global option [Node Drain Policy](https://longhorn.io/docs/1.6.0/references/settings/#node-drain-policy) to control `node drain` strategy.

(1) (2) touch `StorageClass`, (3)~(6) touch LH internal mechanism;  they have the connection via a global setting `Node Drain Policy`.

---

Back to here, suppose a reader has some k8s backgrounds, he reads this block of text, and click the link to troubleshooting section, he will know what we try to descibe.